### PR TITLE
Simple station goals

### PR DESCRIPTION
**Pull request by Morb0**
## About the PR 
Simple yml based realisaion of goals for station. A paper with text of goal is printed at beginning of round on all 
 communication consoles. There is also a command for admin to replace goal to a random or specific by ID.
 
`RoundStartedEvent` has been added, indicating that the map is fully loaded.

**Screenshots**

**Changelog**

:cl: Morty
- add: Each shift now has a goal to complete

### CODE REVIEW

File: Resources/Locale/en-US/station-goals/goals/shuttle.ftl [hunk 1]
```diff
@@ -0,0 +1,2 @@
+﻿station-goal-shuttle-text = The goal of your shift is to build a manned shuttle in space and provide it with the necessities of life.
+                            To do this, you will need cargos to order the necessary components for the shuttle, the work of the engineering department to build it, and the science department to supply the engineers and shuttle with the necessary equipment.
```
Comment (moonheart08) [inline Resources/Locale/en-US/station-goals/goals/shuttle.ftl:2]:
```suggestion
                            To do this, you will need cargo to order the necessary components for the shuttle, the work of the engineering department to build it, and the science department to supply the engineers and shuttle with the necessary equipment.
```

File: Resources/Locale/en-US/station-goals/goals/singularity.ftl [hunk 2]
```diff
@@ -0,0 +1,2 @@
+﻿station-goal-singularity-text = The goal of your shift is to build an additional generator based on the Singularity.
+                                To do this you will need the work of the engineering department to build the "cage" to contain the Singularity, as well as the Cargo department to order the necessary components to build it.
```
Comment (moonheart08) [inline Resources/Locale/en-US/station-goals/goals/singularity.ftl:2]:
```suggestion
                                To do this you will need the work of the engineering department to build the Singularity's containment field, as well as the Cargo department to order the necessary components to build it.
```

**Review by moonheart08 — APPROVED** [review]

**Comment by mirrorcult** [comment]
code looks fine, not sure how i feel about it in concept though

**Comment by Fishfish458** [comment]
Don't mean to be piling on what mirror mentioned earlier, just thoughts. I think station goals work best as something that you're assumed to be doing along the shift, keeping the station maintained, keeping it clean, keeping enough food on the table, serving drinks, keeping people healthy and alive.

Objectives like building building a shuttle can be a bit counter-intuitive since it can take engineers away from doing their actual job to accomplish a goal where they might isolate and leave the station altogether. 

I think a better goal would be determining those job factors listed above at round end and grading the department based off it.

**Comment by Morb0** [comment]
Goals do not have to be as I specified, I just chose the most frequent activities of the players when they get bored. Also goals are aimed at a more dense interaction of departments, gives work for command and shows when to call shuttle.

**Comment by metalgearsloth** [comment]
I agree with mirror.

Realistically I think station events / dynamic does it better as goals for stuff to deal with during a shift.

**Pull request closed without merging**